doc/manual.rst

```diff
@@ -565,6 +565,11 @@ The type suffixes are:
   ``'f64``           float64
 =================    =========================
 
+If the suffix is not predefined, then the suffix is assumed to be a call
+to a proc, template, macro or other callable identifier that is passed the
+string containing the literal. Essentially,  ``1234.56'sample`` is
+the equivalent of ``sample("1234.56")``.
```

Comment:
use `-128'i8custom` as example, it's more interesting as it illustrate something the current parser has a problem with (https://github.com/timotheecour/Nim/issues/125 -128'i8 gives: Error: number out of range) and that user defined literals could fix

Reply:
It's not a parser problem, it's.a spec problem: Unary minus is always separate and never part of a literal. This should be fixed within this PR indeed.

Reply:
Made it easier to read by putting an example block of code that makes a crude `nibble`, a 4-bit uint that doesn't natively exist.